docs/06-environments/04-mainnet.md

```diff
@@ -57,12 +88,19 @@ The following details a list of resources to support the Mainnet environment. EV
 
 ### Registered routes
 
-| Source Network Name | Destination Network Name | Resource | Fee Percent/Amount | Resource ID |
-| ------------------- | ------------------------ | -------- | ------------------ | ---------- |
-| Ethereum Mainnet    | Phala                    | PHA      | 0.0001 ETH         | 0x0000000000000000000000000000000000000000000000000000000000000001  |
+| Source Network Name | Destination Network Name | Resource | Fee Percent/Amount | Resource ID                                                        |
+| ------------------- | ------------------------ | -------- | ------------------ | ------------------------------------------------------------------ |
+| Ethereum Mainnet    | Phala                    | PHA      | 0.0001 ETH         | 0x0000000000000000000000000000000000000000000000000000000000000001 |
 | Ethereum Mainnet    | Khala                    | PHA      | 0.0001 ETH         | 0x0000000000000000000000000000000000000000000000000000000000000001 |
 | Khala               | Ethereum Mainnet         | PHA      | 50 PHA             | 0x0000000000000000000000000000000000000000000000000000000000000001 |
 | Phala               | Ethereum Mainnet         | PHA      | 50 PHA             | 0x0000000000000000000000000000000000000000000000000000000000000001 |
+| Ethereum            | Base                     | GMP      | 0.0008 ETH         |                                                                    |
```

Comment:
Resource ID for GMP is ` 0x0000000000000000000000000000000000000000000000000000000000000000`